common-custom-user-data-gradle-plugin/src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -188,10 +188,21 @@ private void captureCiMetadata() {
             envVariable("BITRISE_BUILD_NUMBER").ifPresent(value ->
                 buildScan.value("CI build number", value));
         }
+
+        if (isGoCD()) {
+            envVariable("GO_SERVER_URL").ifPresent(url ->
+                buildScan.link("GoCD", url));
+            envVariable("GO_PIPELINE_NAME").ifPresent(value ->
+                addCustomValueAndSearchLink("CI pipeline name", value));
```

Comment:
I think `addCustomValueAndSearchLink("CI pipeline", value))` would be better here. 
This will result in:
- Custom value `CI pipline`
- Link `CI pipeline build scans`

(It's also possible to specify the link label separately from the custom value name if required).